include/tunnler/room_member.h

```diff
@@ -86,13 +90,13 @@ class RoomMember final {
      * Sends a chat message to the room.
      * @param message The contents of the message.
      */
-    void SendChatMessage(const std::string message);
+    void SendChatMessage(const std::string& message);
 
 	/**
      * Sends a WiFi packet to the room.
      * @param packet The WiFi packet to send.
      */
-    void SendWifiPacket(const WifiPacket& packet);
+    void SendWifiPacket(WifiPacket& packet);
```

Comment:
I'd do the casting back and forth or even create a copy of the packet, just to not give in to RakNet shenannigans.

Before sending that E-Mail I might check if there is a PR / reason for this and tell them to fix it too :P